.env.development

```diff
@@ -20,3 +20,4 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 SEGMENT_KEY=''
 SITE_NAME=localhost
 USER_INFO_COOKIE_NAME='edx-user-info'
+FEEDER_PROJECT_ID='61f123796593f20004018b2f'
```

Comment:
you shouldn't put API keys here, these should be carried over using env settings in edx internals for MFEs. please remove these.

Reply:
Sure. I'll remove it from this MEF.